presto-docs/src/main/sphinx/release/release-0.262.rst

```diff
@@ -0,0 +1,35 @@
+=============
+Release 0.262
+=============
+
+**Highlights**
+==============
+* Fix the issue of queries getting stuck in the queued state in disaggregated coordinator cluster.
+* Add support for simple TTL based scheduling based on worker availability estimates and query execution time estimates.
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix :func:`reduce_agg` to allow ``NULL`` as the result of input/combine functions, while also allowing only constant expressions and not ``NULL`` as the initial value.
+* Fix the issue of queries getting stuck in the queued state in disaggregated coordinator cluster.
+* Add default size limit (100MB) to build side of broadcast join.
+* Add support for simple TTL based scheduling where the scheduler picks workers for query execution based on their availability estimates and
+  query execution time estimates. This can be enabled by setting either the ``resource_aware_scheduling_strategy`` session property or the
+  ``experimental.resource-aware-scheduling-strategy`` configuration property to ``TTL``. See :pr:`16409`.
+* Add support to read adjusted queue size information from the ``v1/cluster`` endpoint.
+
+SPI Changes
+___________
+* Add support for custom Node TTL fetchers and Cluster TTL providers through the ``NodeTtlFetcher`` and ``ClusterTtlProvider`` interfaces respectively. See :pr:`16409`.
+
+Hive Changes
+____________
+* Fix issue while reading Apache HUDI tables when PrestoFileSystemCache is refreshed (:pr:`16611`).
```

Comment:
Can we explain more on the issue? Especially the `PrestoFileSystemCache` part. If it is not user-facing, maybe we should explain it better.

Reply:
@imjalpreet Can you please update the release notes of https://github.com/prestodb/presto/pull/16611 to address this comment? Thanks!

Reply:
double quotes for PrestoFileSystemCache

Reply:
@neeradsomanchi @mayankgarg1990 PFB the updated release note, let me know if we want to make any change.

```
* Fix issue while reading Apache HUDI tables when ``PrestoFileSystemCache`` is refreshed. After ``PrestoFileSystemCache`` got refreshed, since ``HoodieROTablePathFilter`` was cached with default ``Configuration`` object, it resulted in issues while accessing filesystem (:pr:`16611`)
```